node/p2p/blossomsub.go

```diff
@@ -738,17 +736,20 @@ func initDHT(
 		bootstrapPeerIDs[peerinfo.ID] = struct{}{}
 	}
 	go func() {
+		const minBootstrapPeers = 3
 		for {
 			time.Sleep(30 * time.Second)
-			found := false
+			count := 0
 			for _, p := range h.Network().Peers() {
 				if _, ok := bootstrapPeerIDs[p]; ok {
-					found = true
```

Comment:
The beacon was going to be 'the one bootstrap peer' that we would be connected to quite often, and this meant that we wouldn't attempt to reconnect to the bootstrappers if we had at least the beacon. As such, we now require at least 3 bootstrap peers.